bw2io/strategies/ecospold2.py

```diff
@@ -197,16 +596,64 @@ def delete_ghost_exchanges(db):
 
 
 def remove_uncertainty_from_negative_loss_exchanges(db):
-    """Remove uncertainty from negative lognormal exchanges.
-
-    There are 15699 of these in ecoinvent 3.3 cutoff.
```

Comment:
Is this information provided somewhere? Should be added to 'Notes' section of docstring.

Reply:
resolved